src/js/components/project-view/empty-project.tsx

```diff
@@ -41,7 +41,7 @@ const EmptyProject = ({ project }: Props) => {
                 </pre>
               </div>
               <div className={styles.text}>
-                Add Minard as a remote:
+                Add <code>minard</code> as a remote repository:
```

Comment:
A "remote" is a Git term by itself. It's different than a "remote repository", e.g.: 
> Minard, a **remote repository**, has been set as on of the **remotes** of this repository.